stmt/stmts/update.go

```diff
@@ -279,7 +296,7 @@ func (s *UpdateStmt) Exec(ctx context.Context) (_ rset.Recordset, err error) {
 			end := start + len(tbl.Cols())
 			data := rowData[start:end]
 			start = end
-			tcols, err2 := getUpdateColumns(tbl, s.List, isMultipleTable)
+			tcols, err2 := getUpdateColumns(tbl, s.List, isMultipleTable, tblAliasMap)
```

Comment:
Yes, but we have to do some update in field package to support these,
so i think it is better in a new PR.